.cargo/config.toml

```diff
@@ -15,6 +15,9 @@ rustflags = [
 	"-Welided_lifetimes_in_paths",
 	"-Wunused_extern_crates",
 	"-Wexplicit_outlives_requirements",
+	# Cargo
+	"-Wclippy::cargo",                   # Warn about Cargo.toml issues, except...
```

Comment:
nit: suggest moving below under `clippy` section.

Reply:
Done in https://github.com/NomicFoundation/slang/pull/704/commits/68dbccaeb5c48ec4d8f2d426c28de81667225989